source/dub/commandline.d

```diff
@@ -173,9 +173,29 @@ struct CommandLineHandler
 				// Use default determined in internal.logging.initLogging().
 				break;
 			case on:
+                        	foreach (ref commandGroup; commandGroups)
+                                {
+                                	foreach (ref commands; commandGroup.commands)
+                                        {
```

Comment:
you're mixing tabs and spaces here. Also please dedent by 3 tabs worth and align the braces with the foreach (or drop them entirely). Ditto throughout

Reply:
Resolved at 69ac9438.